boards/BMS/Src/App/states/App_PreChargeState.c

```diff
@@ -20,11 +49,67 @@ static void
     PreChargeStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
     App_AllStatesRunOnTick100Hz(state_machine);
+
+    struct BmsWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct Clock *   clock = App_BmsWorld_GetClock(world);
+    struct Airs *    airs  = App_BmsWorld_GetAirs(world);
+
+    float tractive_system_voltage = Io_VoltageSense_GetTractiveSystemVoltage();
+    float accumulator_voltage     = 355.0f; // TODO: Replace value with function to
+                                        // retrieve total pack voltage
+    uint32_t elapsed_time =
```

Comment:
dont expect this to change so nicer if we make it a const

```suggestion
    const uint32_t elapsed_time =
```

Reply:
this changes every 100Hz

Reply:
yup thats right but when this function is getting called its not changing so its nice to keep it as a const